ledger/participant-integration-api/src/main/scala/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -121,8 +123,8 @@ final class StandaloneApiServer(
         servicesExecutionContext,
         metrics,
       )
+      _ <- new FutureResourceOwner[ResourceContext, Unit](() => writePortFile(apiServer.port))
```

Comment:
This could be:

```scala
      _ <- ResourceOwner.forFuture(() => writePortFile(apiServer.port))
```

Reply:
@SamirTalwar-DA Thank you, that's a good suggestion. I've implemented the `forTry` approach [here](https://github.com/digital-asset/daml/pull/9720).